### DISCUSSION TITLE
Disable 'Featured Notebooks'

### DISCUSSION THREAD

**Discussion by jeremiahpslewis** [fonsp/Pluto.jl#2375] [Questions about using Pluto] [answered]
The new 'Featured Notebooks' section is a great addition, especially for new users, but adds a lot of visual noise if you are familiar with Pluto and use it regularly. Is it be possible to turn it off via a config flag?

**Comment by fonsp** [comment, marked as answer]
Currently not possible, sorry! But you can fork of course.
